src/app/components/components.component.ts

```diff
@@ -71,8 +71,8 @@ export class ComponentsComponent implements OnInit, OnDestroy {
         }
 
         this.allDocItems = this.docItemService.getDocItems(this.docType);
-        if (!this.id) {
-            this.navUpdate(this.allDocItems[0].id);
+        if (!this.id && this.categorizedDocItems[0] && this.categorizedDocItems[0].items && this.categorizedDocItems[0].items.length) {
```

Comment:
Ran into an edge case where it navigated to `/components/undefined/examples`... this should prevent that.